GoogleChrome/lighthouse
# report: update error icons

**Summary**


Fixes #9820 


Updates runtime error icons to use the SVGs provided in https://github.com/GoogleChrome/lighthouse/issues/9820#issuecomment-541796558

<img width="575" alt="Screen Shot 2023-05-17 at 11 30 28 PM" src="https://github.com/GoogleChrome/lighthouse/assets/48612525/62a98940-d43e-4c54-a41c-fd39e45f56aa">



**Testing**
- run `yarn build-sample-reports`
- view the .cdt.error/index.html or .psi.error/index.html sample reports
- see the updated runtime error icon

**Related Issues/PRs**

https://github.com/GoogleChrome/lighthouse/pull/9272
https://github.com/GoogleChrome/lighthouse/issues/9968#issuecomment-675704398
